watertap/property_models/multicomp_aq_sol_prop_pack.py

```diff
@@ -2270,15 +2272,21 @@ def calculate_scaling_factors(self):
 
         if self.is_property_constructed("ionic_strength_molal"):
             if iscale.get_scaling_factor(self.ionic_strength_molal) is None:
-                sf = min(
-                    iscale.get_scaling_factor(self.molality_phase_comp["Liq", j])
-                    for j in self.params.solute_set
+                sf = (
+                    sum(
+                        iscale.get_scaling_factor(self.molality_phase_comp["Liq", j])
+                        ** -1
+                        * iscale.get_scaling_factor(self.charge_comp[j]) ** -1
+                        for j in self.params.solute_set
+                    )
+                    ** -1
+                    * 2
                 )
                 iscale.set_scaling_factor(self.ionic_strength_molal, sf)
 
         if self.is_property_constructed("total_hardness"):
             if iscale.get_scaling_factor(self.total_hardness) is None:
-                sf = 10 / value(self.total_hardness)
+                sf = 1/100
```

Comment:
I think this may have been replaced by a conditional that checks if `total_hardness` is 0, which might've gotten merged first. You can see there is a merge conflict though because of it, so wanted you to check this over.

Reply:
yup resolved this just now, lets see if all tests pass, if so can you approve it so we can merge this in!